addons/web/static/src/js/views/graph/graph_view.js

```diff
@@ -155,6 +156,15 @@ var GraphView = AbstractView.extend({
         this.loadParams.comparisonDomain = params.comparisonDomain;
         this.loadParams.stacked = this.arch.attrs.stacked !== "False";
     },
+
+     /**
+     *
+     * @override
+     */
+    getRenderer(parent, props) {
+        props = Object.assign(props || {}, this.rendererParams);
```

Comment:
We called it `state` in the other views, but I think you're right to name it `props` :'(